### ISSUE TITLE
error after single-writer multiple-reader deployment

### ISSUE THREAD

**Issue by caseyjlaw** [lwa-project/ovro_data_recorder#40]
The solar team is running the dr2 recorder on lwacalim01. They report that the beams produce data that look like they are not tracking the sun. From what I can tell, the scheduler (outside of the ovro-data-recorder code), is issuing commands and tracking as expected.

Looking at the dr2 logs, I see an error after starting recording:
```
2023-11-01 15:08:48 [INFO    ] HDF5Record - Scheduled recording for 2023-11-01 15:09:03.380554 to 2023-11-01 15:09:13.380554 to /data01/beam02/060249_1509033805548f0b56c
2023-11-01 15:09:04 [INFO    ] Started operation - <HDF5Writer filename='/data01/beam02/060249_1509033805548f0b56c',
    start_time='2023-11-01 15:09:03.380554', stop_time='2023-11-01
    15:09:13.380554', reduction=<ReductionOperation time_avg=1,
    chan_avg=1, pols='XX,YY,CR,CI'>>
2023-11-01 15:09:14 [ERROR   ] Write error asserted - initial error: 'int' object has no attribute 'flush'
2023-11-01 15:09:16 [INFO    ] Ended operation - <HDF5Writer filename='/data01/beam02/060249_1509033805548f0b56c',
    start_time='2023-11-01 15:09:03.380554', stop_time='2023-11-01
    15:09:13.380554', reduction=<ReductionOperation time_avg=1,
    chan_avg=1, pols='XX,YY,CR,CI'>>
```

Could that be related to the report of a beam with data that is not tracking?

**Comment by jaycedowell** [comment]
This would probably stop data from being written to disk but that is independent of whether or not the beam is tracking.

**Comment by jaycedowell** [comment]
This error should be fixed with [6c20b13](https://github.com/lwa-project/ovro_data_recorder/commit/6c20b130502d44940bfa7dd9f926eaa158c82d87).
